katas/proper-parenthetics/src/proper_parenthetics.py

```diff
@@ -0,0 +1,24 @@
+"""Test strings of parens to see if the are open, balanced, or closed."""
+from stack import Stack
+
+
+def proper_parenthetics(parens):
+    """Function to check open and close parens."""
+    if type(parens) != str:
+        raise TypeError('This must be a string.')
+    paren_list = Stack(parens[::-1])
+    open_paren = 0
+    close_paren = 0
+    pops = 0
+    while pops != len(parens):
```

Comment:
What if `parens` is an empty string? Your `pop` method below is going to throw an error.